### PR TITLE
Expose Sync API On SyncAsyncFileSystemDecorator.js

### LINKED ISSUE #147: Export Sync methods in SyncAsyncFileSystemDecorator.js
Hi!

I've noticed that `enhanced-resolve/lib/SyncAsyncFileSystemDecorator.js` did not expose the Sync API.
I wanted to ask you if you would consider a PR that would add something along those lines :
```js
this.statSync = fs.statSync;
this.readdir = fs.readdir;
this.readFile = fs.readFile;
this.readlink = fs.readlink;
this.readJson = fs.readJson;
```

The only negative impact that I see is that it wouldn't motivate people to use the async API in most cases. But I guess we can add a wrapper around each function that prints a warning.

### PR DESCRIPTION
**Pull request by samolabams**
resolve #147 - I added the SYNC API  on **enhanced-resolve/lib/SyncAsyncFileSystemDecorator.js**. According to issue #147

### CODE REVIEW

**Comment by samolabams** [comment]
I will add tests

File: lib/SyncAsyncFileSystemDecorator.js [hunk 1]
```diff
@@ -16,6 +16,16 @@ function SyncAsyncFileSystemDecorator(fs) {
 			}
 			callback(null, result);
 		};
+
+		this.statSync = function(arg) {
```
Comment (sokra) [inline lib/SyncAsyncFileSystemDecorator.js]: This is unneccessarily complex. You can write it as:

``` js
this.statSync = function(arg) {
  return fs.statSync(arg);
}
```

for all of them

Comment (samolabams) [reply lib/SyncAsyncFileSystemDecorator.js]: Okay, i will make those changes

**Comment by evenstensberg** [comment]
@evilebottnawi could you elaborate on where to write the test and what the test case should cover?

**Comment by sokra** [comment]
merged

**Comment by evenstensberg** [comment]
@sokra do you have a commit reference of this?

**Pull request closed without merging**